projects/observability/src/pages/apis/services/services-list.dashboard.ts

```diff
@@ -1,194 +1,129 @@
-      width: '30%',
-      value: {
-        type: 'entity-specification'
-      }
-    },
-    {
-      type: 'table-widget-column',
-      title: 'Service Name',
-      width: '30%',
-      value: {
-        type: 'attribute-specification',
-        attribute: 'serviceName'
-      }
-    },
-    {
-      type: 'table-widget-column',
-      title: 'p99 Latency',
-      display: TracingTableCellType.Metric,
-      value: {
-        type: 'metric-aggregation',
-        metric: 'duration',
-        aggregation: 'p99'
-      },
-      sort: TableSortDirection.Descending
-    },
-    {
-      type: 'table-widget-column',
-      title: 'Avg Latency',
-      display: TracingTableCellType.Metric,
-      value: {
-        type: 'metric-aggregation',
-        metric: 'duration',
-        aggregation: 'avg'
-      }
-    },
-    {
-      type: 'table-widget-column',
-      title: 'Errors/s',
-      display: CoreTableCellRendererType.Number,
-      value: {
-        type: 'metric-aggregation',
-        metric: 'errorCount',
-        aggregation: 'avgrate_sec'
-      }
-    },
-    {
-      type: 'table-widget-column',
-      title: 'Errors',
-      display: CoreTableCellRendererType.Number,
-      value: {
-        type: 'metric-aggregation',
-        metric: 'errorCount',
-        aggregation: 'sum'
-      }
-    },
-    {
-      type: 'table-widget-column',
-      title: 'Calls/s',
-      display: CoreTableCellRendererType.Number,
-      value: {
-        type: 'metric-aggregation',
-        metric: 'numCalls',
-        aggregation: 'avgrate_sec'
-      }
-    },
-    {
-      type: 'table-widget-column',
-      title: 'Calls',
-      display: CoreTableCellRendererType.Number,
-      value: {
-        type: 'metric-aggregation',
-        metric: 'numCalls',
-        aggregation: 'sum'
-      }
-    }
-  ],
-  data: {
-    type: 'entity-table-data-source',
-    entity: 'API'
-  }
-};
+import { ObservabilityEntityType } from '../../../shared/graphql/model/schema/entity';
 
 export const servicesListDashboard: DashboardDefaultConfiguration = {
   location: 'SERVICE_LIST',
   json: {
-    type: 'mode-toggle-table-widget',
-    searchAttribute: 'name',
-    style: TableStyle.FullPage,
-    mode: TableMode.Tree,
-    modeOptions: [TableMode.Tree, TableMode.Flat],
-    flat: flatTableWidget,
-    tree: treeTableWidget
+    type: 'container-widget',
+    layout: {
+      type: 'auto-container-layout',
+      'enable-style': false
+    },
+    children: [
+      {
+        type: 'table-widget',
+        id: 'services-list.table',
+        mode: TableMode.Flat,
+        style: TableStyle.FullPage,
+        searchAttribute: 'name',
+        selectFilterOptions: [
+          {
+            type: 'table-widget-select-filter',
+            attribute: 'serviceName',
+            'unset-option': 'All Services',
+            data: {
+              type: 'entities-attribute-data-source',
+              entity: ObservabilityEntityType.Api,
```

Comment:
Given the change in the attribute data source, this can change to SERVICE.name right? (the main reason I'm pushing down this path is actually the scale of api data - there could be thousands, so fetching only the services seems much more efficient and less likely to hit the hardcoded limit I think I saw - 100, right?)